lib/protocol/extra.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Protocol.Extra do
+  @moduledoc """
+  Extensions to the standard library's Protocol module.
+
+  """
+
+  @doc """
+  Returns true if the given *module* defines a struct which implements the given protocol.
+
+  ## Examples
+
+      iex> Protocol.Extra.module_implements?(Enumerable, Stream)
+      true
+
+      iex> Protocol.Extra.module_implements?(String.Chars, Stream)
+      false
+
+  """
+  @spec module_implements?(Protocol.t, module) :: boolean
+  def module_implements?(protocol, struct_module)
```

Comment:
Thoughts on renaming to impl_by_module? - seems like a better name since the protocol is the first argument

Reply:
1. I dislike abbreviating in public names, so if anything it'd be "implemented_by_module"
2. passive voice is :-1: